doc/source/ray-core/handling-dependencies.rst

```diff
@@ -429,11 +429,12 @@ Are environments installed on every node?
 """""""""""""""""""""""""""""""""""""""""
 
 If a runtime environment is specified in ``ray.init(runtime_env=...)``, then the environment will be installed on every node.  See :ref:`Per-Job <rte-per-job>` for more details.
+(Note, by default the runtime environment will be installed eagerly on every node in the cluster. If you want to lazily install the runtime environment on demand, set the ``eager_install`` option to false: ``ray.init(runtime_env={..., "eager_install": True}``.)
```

Comment:
```suggestion
(Note, by default the runtime environment will be installed eagerly on every node in the cluster. If you want to lazily install the runtime environment on demand, set the ``eager_install`` option to false: ``ray.init(runtime_env={..., "eager_install": False}``.)
```

Reply:
How about we leave this in the `When is the environment installed?` section and add the detail there instead? Seems like the most logical place instead of duplicating the information around

Reply:
My concern is that maybe some users only read the section `Are environments installed on every node?` and they may misunderstand that the job level runtime_env can only be installed on all nodes eagerly. Wdyt?

Reply:
Actually, I think as-is is good otherwise it's less of a FAQ...